source/get-started/download-and-install.txt

```diff
@@ -24,12 +24,13 @@ Download and Install
       - `CMake <https://cmake.org/>`__ v3.15 or later
       - `pkg-config <https://www.freedesktop.org/wiki/Software/pkg-config/>`__
 
-      .. note:: C++17 Polyfills
+      .. note:: Pre-C++17 Configurations
```

Comment:
The "macos / Linux" CMake configuration command is missing `-DCMAKE_CXX_STANDARD=17` to select the C++17 standard as implied by this note.

This page also states "Compiler that supports C++17" as a requirement above despite the minimum supported C++ standard being C++11. This page may need to be reworded to document C++11 as the minimum requirement, but instead strongly recommend users configure the driver with `CMAKE_CXX_STANDARD=17` instead when able.

Reply:
Added the `-DCMAKE_CXX_STANDARD=17` option! Since this tutorial does set those CMake options and configures the driver to use C++17, I think "Compiler that supports C++17" should still be a prerequisite. I reworded that step to make it clear that this is a requirement for the tutorial, rather than for configuring the driver in general.